server/resp/src/main/java/org/infinispan/server/resp/serialization/bytebuf/ByteBufCollectionSerializer.java

```diff
@@ -56,8 +56,8 @@ public void accept(Collection<?> objects, ByteBufPool alloc, SerializationHint.S
 
       @Override
       public boolean test(Object object) {
-         // Handle any collection which is not a set.
-         return object instanceof Collection<?> && !(object instanceof Set<?>);
+         // Handle any collection
+         return object instanceof Collection<?>;
```

Comment:
Is this change needed? I am a bit surprised at the change as this deals with the ByteBuf serializer..